Tests/Data/PhreeqcDatabase/readme.txt

```diff
@@ -0,0 +1 @@
+The thermodynamic database files included in this folder come from the download package of IPhreeqc module (iphreeqc-3.5.0-14000) which can be accessed via https://water.usgs.gov/water-resources/software/PHREEQC/iphreeqc-3.5.0-14000.tar.gz
```

Comment:
Please add the license to the readme:
```
Additional information for the iphreeqc software is available on their website https://www.usgs.gov/software/phreeqc-version-3/

The license of the phreeqc databases is:

Software License and Purchase Information
This software is a product of the U.S. Geological Survey, which is part of the U.S. Government.

Cost

This software is freely distributed. There is no fee to download and (or) use this software.

License

Users do not need a license or permission from the USGS to use this software. Users can download and install as many copies of the software as they need.

Public Domain

As a work of the United States Government, this USGS product is in the public domain within the United States. You can copy, modify, distribute, and perform the work, even for commercial purposes, all without asking permission. Additionally, USGS waives copyright and related rights in the work worldwide through CC0 1.0 Universal Public Domain Dedication (https://creativecommons.org/publicdomain/zero/1.0/ ).
```

Reply:
or even better, to include the `NOTICE` file from the `iphreeqc-3.5.0-14000/doc` folder.

Reply:
Added.